src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/trial_of_the_champion.h

```diff
@@ -15,124 +16,284 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef DEF_TOC_H
-#define DEF_TOC_H
+#ifndef TRIAL_OF_CHAMPION_H_
+#define TRIAL_OF_CHAMPION_H_
```

Comment:
This change looks outdated, is it intentional?

Reply:
Never mind, I was thinking of something else entirely. (I will delete these 2 comments.)